hathor/feature_activation/feature.py

```diff
@@ -33,3 +33,5 @@ class Feature(Enum):
     NOP_FEATURE_4 = 'NOP_FEATURE_4'
     NOP_FEATURE_5 = 'NOP_FEATURE_5'
     NOP_FEATURE_6 = 'NOP_FEATURE_6'
+
+    INCREASE_MAX_MERKLE_PATH_LENGTH = 'INCREASE_MAX_MERKLE_PATH_LENGTH'
```

Comment:
Should we prefix it with `TESTNET_`?

Reply:
We could if you see any advantage, however it's designed to not be necessary. This is simply a "feature identifier", and defining it here doesn't enable anything. To actually configure the process, this identifier has to be referenced in the network's configuration file. In other words, this can be reused for both testnet and mainnet, even if they use completely separate processes.

Reply:
Got it. So it makes sense to not use it unless it's something exclusive for a network. Thanks!